src/cdk/drag-drop/directives/drag-handle.ts

```diff
@@ -18,6 +18,7 @@ import {
   SkipSelf,
 } from '@angular/core';
 import {Subject} from 'rxjs';
+import type {CdkDrag} from './drag';
```

Comment:
If I remember correctly, we aren't allowed to use type imports because the internal JS tooling can't handle them.

Reply:
I've confirmed that we can't use type-only imports and exports, because Closure doesn't know how to handle them. I've also submitted #24678 to lint against it automatically.